src/CalculatorApi.h

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
```

Comment:
You need include guards on .h files. Include guards help in the following scenario: say you have a.h, b.h, and c.h. Say a.h includes b.h and c.h. Say b.h includes c.h. In this scenario, without include guards, the contents of c.h get copied into b.h, which then gets copied into a.h, but c.h is also included by a.h, so c.h gets copied into a.h again (twice, once indirectly from b.h and once directly from c.h).

You want to start any .h file with something like:
```
#ifndef CALCULATORAPI_H
#define CALCULATORAPI_H
```
and end it with
`#endif // #ifndef CALCULATORAPI_H`
This makes sure that the file will only be included once, then that `CALCULATORAPI_H` gets defined. If it tries to get included into the same file again, the `#ifndef CALCULATORAPI_H` will not be true, since `CALCULATORAPI_H` got defined.